evalai/utils/challenges.py

```diff
@@ -160,3 +160,106 @@ def get_challenge_count(is_host=False, is_participant=False):
 
     for challenge in challenges:
         print_challenge_table(challenge)
+
+
+def get_phase_list(challenge_id):
+    """
+    Gets the phase lists of a particular challenge.
+    """
+    url = Urls.phase_list.value
+    url = "{}{}".format(API_HOST_URL, url)
+    url = url.format(challenge_id)
+    headers = get_headers()
+
+    try:
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        echo(err)
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(err)
+        sys.exit(1)
+
+    response_json = response.json()
+    if valid_token(response_json):
+        phases = response_json["results"]
+        for phase in phases:
+            br = style("--------------------------------"
```

Comment:
This section should go in a separate `print_phase` function that can be easily updated later.